src/libsync/networkjobs.cpp

```diff
@@ -636,14 +613,15 @@ bool AvatarJob::finished()
 #endif
 
 /*********************************************************************************************/
-EntityExistsJob::EntityExistsJob(AccountPtr account, const QString &path, QObject *parent)
-    : AbstractNetworkJob(account, path, parent)
+
+EntityExistsJob::EntityExistsJob(AccountPtr account, const QUrl &rootUrl, const QString &path, QObject *parent)
+    : AbstractNetworkJob(account, rootUrl, path, parent)
 {
 }
 
 void EntityExistsJob::start()
 {
-    sendRequest("HEAD", makeAccountUrl(path()));
+    sendRequest("HEAD");
```

Comment:
Since the URL is now considered essential data and passed in the constructor, the keyword should likely be managed the same way. Unless there is a job which uses multiple (but then I guess there's a violation of the SRP).